src/validateRequest.js

```diff
@@ -0,0 +1,33 @@
+const { SUPPORTED_CASES } = require('./constants');
+
+function validateRequest(textToConvert, targetCase) {
+  const errors = [];
+
+  if (!textToConvert) {
+    errors.push({
+      message:
+        'Text to convert is required.'
+        + ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
```

Comment:
Better to move all these messages to file with constants.
Also this part `' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".'` is the same in 2 messages